Core/StartupArgManager.cs

```diff
@@ -14,35 +14,32 @@ public static class StartupArgManager
         private const string StartupArgsPath = "laargs.txt";
```

Comment:
Can this be changed to the config folder of the SL AppData? This may avoid any read/write issues and keeps all the other config files.

Reply:
That's out of scope of this PR and the path wasn't changed by this PR. If you wish make a PR changing the path or open an issue.

Btw. it's your code :)